README.md

```diff
@@ -103,57 +103,126 @@ FLP is a framework. The main FLP object is the **pipeline**. FLP **pipeline** ca
+Multiple stages may follow from a particular stage, thus allowing the same data to be fed into multiple parallel pipelines.
+Multiple **transform** stages may be performed and the results may be output to different targets.
 
+A full configuration file with the data fed into two different transforms might look like the following.
+
+```yaml
+pipeline:
+  - name: ingest1
+  - name: decode1
+    follows: ingest1
+  - name: generic1
+    follows: decode1
+  - name: write1
+    follows: generic1
+  - name: generic2
+    follows: decode1
+  - name: write2
+    follows: generic2
+parameters:
+  - name: decode1
+    decode:
+      type: json
+  - name: generic1
+    transform:
+      type: generic
+      generic:
+        rules:
+          - input: Bytes
+            output: v1_bytes
+          - input: DstAddr
+            output: v1_dstAddr
+          - input: Packets
+            output: v1_packets
+          - input: SrcPort
+            output: v1_srcPort
+  - name: write1
+    write:
+      type: stdout
+  - name: generic2
+    transform:
+      type: generic
+      generic:
+        rules:
+          - input: Bytes
+            output: v2_bytes
+          - input: DstAddr
+            output: v2_dstAddr
+          - input: Packets
+            output: v2_packets
+          - input: SrcPort
+            output: v2_srcPort
+  - name: ingest1
+    ingest:
+      type: file_loop
+      file:
+        filename: playground/goflow2_input.txt
+  - name: write2
+    write:
+      type: stdout
+```
 It is expected that the **ingest** module will receive flows every so often, and this ingestion event will then trigger the rest of the pipeline. So, it is the responsibility of the **ingest** module to provide the timing of when (and how often) the pipeline will run.
 
 # Configuration
 
 It is possible to configure flowlogs-pipeline using command-line-parameters, configuration file, environment variables, or any combination of those options.
 
+
 For example:
 1. Using command line parameters:
-```./flowlogs-pipeline --pipeline.ingest.type file --pipeline.ingest.file.filename hack/examples/ocp-ipfix-flowlogs.json```
+```./flowlogs-pipeline --log-level info ```
 2. Using configuration file:
 - create under $HOME/.flowlogs-pipeline.yaml the file:
 ```yaml
+log-level: info
 pipeline:
-  ingest:
-    type: file
-    file:
-      filename: hack/examples/ocp-ipfix-flowlogs.json
+  - name: ingest_file
+  - name: decode_json
+    follows: ingest_file
+parameters
+  - name ingest_file
+    ingest:
+      type: file
+      file:
+        filename: hack/examples/ocp-ipfix-flowlogs.json
+  - name: decode_json
+    decode:
+      type: json
 ```
 - execute `./flowlogs-pipeline`
 3. using environment variables:
 - set environment variables
 ```bash
-export FLOWLOGS-PIPELINE_PIPELINE_INGEST_TYPE=file
-export FLOWLOGS-PIPELINE_PIPELINE_INGEST_FILE_FILENAME=hack/examples/ocp-ipfix-flowlogs.json
+export FLOWLOGS-PIPELINE_LOG_LEVEL=info
```

Comment:
same as (1) this will not sore without some environment variables specifying the file

Reply:
The environment variables do not work at present. We are not allowed a `-` (dash) in an environment variable. I will remove this from the current documentation.